.github/workflows/ci.yaml

```diff
@@ -112,35 +111,25 @@ jobs:
         id: login-ecr
         uses: aws-actions/amazon-ecr-login@062b18b96a7aff071d4dc91bc00c4c1a7945b076 # v2.0.1
 
-      - name: Tag Images
-        id: tag_images
-        shell: bash
+      - name: Set Registry Type
+        id: set_registry
         run: |
-          version="${{ steps.get_version.outputs.version }}"
-          release_tag="${{ steps.release_tag.outputs.release_tag }}"
-          short_sha="${{ steps.get_commit_hash.outputs.short_sha }}"
-
-          # Determine image name based on event and branch
+          # Determine if we should use ECR (don't expose the actual registry URL)
           if [[ "${{ github.event_name }}" == "push" && "${{ github.ref_name }}" == "${{ github.event.repository.default_branch }}" ]]; then
-            # Push to default branch - use ECR registry
-            image_name="${{ secrets.SAM_AWS_ECR_REGISTRY }}/solace-agent-mesh"
-            echo "Using ECR registry for default branch push"
+            # Check if ECR registry is available
```

Comment:
the image tag was an output from a previous job to prepare-manifest. The image tag contains a secret (ECR registry) so it cannot be an output. This has to be referred directly to the job that builds the docker image.